From dcb52b0844c21c7fe591ef1cfacafbda1df770f0 Mon Sep 17 00:00:00 2001
From: David Henningsson <david.henningsson@canonical.com>
Date: Fri, 27 Sep 2013 10:22:53 +0200
Subject: [PATCH] default/system.pa: Do not load module-dbus-protocol

The author of this module, Tanu Kaskinen, has said that this module
"is not suitable for general use". Also, it is still causing crashes
on card removal (see bug 69871).

Qpaeq, and possibly other tools, use this module - but they can load
the module manually if they still wish to use it.

Signed-off-by: David Henningsson <david.henningsson@canonical.com>
---
 src/daemon/default.pa.in |    7 -------
 src/daemon/system.pa.in  |    5 -----
 2 files changed, 12 deletions(-)

diff --git a/src/daemon/default.pa.in b/src/daemon/default.pa.in
index f50d929..4e77ae9 100755
--- a/src/daemon/default.pa.in
+++ b/src/daemon/default.pa.in
@@ -171,13 +171,6 @@ load-module module-filter-heuristics
 load-module module-filter-apply
 ])dnl
 
-ifelse(@HAVE_DBUS@, 1, [dnl
-### Load DBus protocol
-.ifexists module-dbus-protocol@PA_SOEXT@
-load-module module-dbus-protocol
-.endif
-])dnl
-
 ifelse(@HAVE_X11@, 1, [dnl
 # X11 modules should not be started from default.pa so that one daemon
 # can be shared by multiple sessions.
diff --git a/src/daemon/system.pa.in b/src/daemon/system.pa.in
index e881a12..6da880e 100755
--- a/src/daemon/system.pa.in
+++ b/src/daemon/system.pa.in
@@ -52,11 +52,6 @@ load-module module-device-restore
 ### that look up the default sink/source get the right value
 load-module module-default-device-restore
 
-.ifexists module-dbus-protocol@PA_SOEXT@
-### If you want to allow TCP connections, set access to "remote" or "local,remote".
-load-module module-dbus-protocol access=local
-.endif
-
 ### Automatically move streams to the default sink if the sink they are
 ### connected to dies, similar for sources
 load-module module-rescue-streams
-- 
1.7.9.5

